src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1025,9 +1020,17 @@ internal AttributeArgumentListSyntax ParseAttributeArgumentList()
                                 {
                                     break;
                                 }
-                                else if (this.CurrentToken.Kind == SyntaxKind.CommaToken || this.IsPossibleAttributeArgument())
+                                else if (this.CurrentToken.Kind == SyntaxKind.CommaToken ||
+                                         this.IsPossibleAttributeArgument())
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?